packages/visx-demo/src/sandboxes/visx-geo-albers-usa/Example.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { AlbersUsa } from '@visx/geo';
+import * as topojson from 'topojson-client';
+import topology from './usa-topo.json';
+import stateAbbrs from './us-abbr.json';
+import { geoCentroid } from 'd3-geo';
+
+export const background = '#EBF4F3';
+
+export type GeoAlbersUsaProps = {
+  width: number;
+  height: number;
+  fullSize: boolean;
+};
+
+interface FeatureShape {
+  type: 'Feature';
+  id: string;
+  geometry: { coordinates: [number, number][][]; type: 'Polygon' };
+  properties: { name: string };
+}
+
+// @ts-ignore
+const { features: unitedStates } = topojson.feature(topology, topology.objects.states) as {
+  type: 'FeatureCollection';
+  features: FeatureShape[];
+};
+
+export const colors: Array<string> = ['#744DCA', '#3D009C', '#9020FF', '#C630FD']
+
+const nudges = {
+  'FL': [11, 3],
+  'AK': [0, -4],
+  'CA': [-7, 0],
+  'NY': [5, 0],
+  'MI': [13, 20],
+  'LA': [-10, -3],
+  'HI': [-10, 10],
+  'ID': [0, 10],
+  'WV': [-2, 4],
+  'KY': [10, 0],
+  'TN': [0, 4]
+}
+
+const GeoAlbersUsa = ({ width, height, fullSize = true }: GeoAlbersUsaProps) => {
+  const centerX = width / 2;
+  const centerY = height / 2;
+  const scale = (width + height) / 1.55;
+
+  const [ displayLabels, setDisplayLabels ] = useState(fullSize)
```

Comment:
this may fail CI, you might want to try running the following locally as I noticed a few missing semi-colons, etc. (from the monorepo root)

```bash
yarn format
yarn lint:fix
```

```suggestion
  const [displayLabels, setDisplayLabels] = useState(fullSize);
```